contracts/token/ERC721_Metadata_base.cairo

```diff
@@ -0,0 +1,42 @@
+%lang starknet
+
+from starkware.cairo.common.cairo_builtins import HashBuiltin, SignatureBuiltin
+from starkware.cairo.common.uint256 import Uint256
+
+from contracts.token.ERC721_base import (
+    ERC721_name_,
+    ERC721_symbol_,
+    ERC721_balanceOf,
+    ERC721_ownerOf,
+    ERC721_getApproved,
+    ERC721_isApprovedForAll,
+    ERC721_tokenURI,
+
+    ERC721_initializer,
+    ERC721_approve, 
+    ERC721_setApprovalForAll, 
+    ERC721_transferFrom,
+    ERC721_safeTransferFrom,
+    ERC721_mint,
+    ERC721_set_baseURI
+)
```

Comment:
I'm undecided whether we want to re-export this from here or do we want the extension to be used independently (but always depending on the user using also 721_base)

Reply:
I'm leaning towards not reexporting, considering users already need to know the dependency chain in order to call initialziers

Reply:
Haha I went back and forth on this for a while. Since 721_base must also be used, I considered this to be easier to replicate ie just grab everything from one source instead of two. I'll refactor and we can reconvene if necessary

Reply:
Yeah not an easy call. Let's keep it simple without re exports.